src/main/features/dashboard/views/macro/claimStatus/paidInFullDefendant.njk

```diff
@@ -11,7 +11,7 @@
 
 {% macro paidInFullDefendantClaimDetails(claim) %}
   <h2 class="heading-small">{{ t('This claim is settled') }}</h2>
-  <p>{{ t('{{ claimantName }} confirmed you paid on {{ moneyReceivedOn }}.',
+  <p>{{ t('{{ claimantName }} confirmed you settled on {{ moneyReceivedOn }}.',
       { claimantName: claim.claimData.claimant.name, moneyReceivedOn: claim.moneyReceivedOn | date }) }}</p>
   <p>{{ downloadResponseLink(claim) }}</p>
 {% endmacro %}
```

Comment:
as discussed, need the other two statuses updated as they are currently wrong in production and it makes sense to fix them and keep them all the same.

Reply:
done